sky/packages/sky_services/BUILD.gn

```diff
@@ -2,33 +2,54 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
-import("//mojo/public/dart/rules.gni")
+if (is_fuchsia) {
+  import("//build/dart/dart_package.gni")
+} else {
+  import("//mojo/public/dart/rules.gni")
+}
 
-copy("copy_sky_services_license") {
-  sources = [
-    "//AUTHORS",
-    "//LICENSE",
-  ]
+if (is_fuchsia) {
+  dart_package("sky_services") {
+    infer_package_name = true
+    deps = [
+      "//mojo/dart/packages/mojo",
+      "//flutter/services/activity:interfaces_dart",
+      "//flutter/services/editing:interfaces_dart",
+      "//flutter/services/engine:interfaces_dart",
+      "//flutter/services/media:interfaces_dart",
+      "//flutter/services/platform:interfaces_dart",
+      "//flutter/services/pointer:interfaces_dart",
+      "//flutter/services/raw_keyboard:interfaces_dart",
+      "//flutter/services/semantics:interfaces_dart",
+    ]
+  }
+} else {
+  copy("copy_sky_services_license") {
+    sources = [
+      "//AUTHORS",
+      "//LICENSE",
+    ]
 
-  outputs = [
-    "$root_gen_dir/dart-pkg/sky_services/{{source_file_part}}",
-  ]
-}
+    outputs = [
+      "$root_gen_dir/dart-pkg/sky_services/{{source_file_part}}",
+    ]
+  }
 
-dart_pkg("sky_services") {
-  sources = [
-    "README.md",
-    "pubspec.yaml",
-  ]
+  dart_pkg("sky_services") {
+    sources = [
+      "README.md",
+      "pubspec.yaml",
+    ]
 
-  deps = [
-    ":copy_sky_services_license",
-    "//flutter/services/activity:interfaces",
-    "//flutter/services/editing:interfaces",
-    "//flutter/services/media:interfaces",
-    "//flutter/services/platform:interfaces",
-    "//flutter/services/pointer:interfaces",
-    "//flutter/services/raw_keyboard:interfaces",
-    "//flutter/services/semantics:interfaces",
-  ]
+    deps = [
+      ":copy_sky_services_license",
```

Comment:
We tar it up ourselves and put it on Google storage.  The requirement is just that we can get a copy of these files onto end-developer machines in standalone Flutter.  We control all pieces of the equation, so it should be possible.

We'll probably also remove many of these interfaces and replace them with JSON message passing with the end goal of removing constraints from the upstream mojo/mojom tooling.